weatherrouting/polar.py

```diff
@@ -189,3 +196,68 @@ def get_twa_routage(self, tws: float, twa: float) -> float:
             if twa > twadown:
                 twa = twadown
         return twa
+
+    @staticmethod
+    def validate_polar_file(filepath):  # noqa: C901
+        try:
```

Comment:
This try catch over the whole function is not a good practice. We need to handle every case inside the body